pkg/actions/action.go

```diff
@@ -84,7 +84,10 @@ func isEventSupported(event graveler.EventType) bool {
 		graveler.EventTypePreDeleteTag,
 		graveler.EventTypePostDeleteTag,
 		graveler.EventTypePreRevert,
-		graveler.EventTypePostRevert:
+		graveler.EventTypePostRevert,
+		graveler.EventTypePreCherryPick,
+		graveler.EventTypePostCherryPick,
```

Comment:
Is this an unrelated bugfix?

Reply:
cc @nopcoder as I see you implemented this in https://github.com/treeverse/lakeFS/pull/9263.

Reply:
it looks like a bug fix for my pr, can take it as a different pr.
and I need to think of a better way to validate next time I'll not miss this one.